Northstar.CustomServers/mod/cfg/autoexec_ns_server.cfg

```diff
@@ -21,4 +21,7 @@ sv_updaterate_mp 20 // default updaterate: 20 tick
 sv_minupdaterate 20 // unsure if this actually works, but if it does, should set minimum client updaterate
 sv_max_snapshots_multiplayer 300 // this needs to be updaterate * 15, or clients will dc in killreplay
 net_data_block_enabled 0 // not really sure on this, have heard datablock could have security issues? doesn't seem to have any adverse effects being disabled
-host_skip_client_dll_crc  1 // allow people to run modded client dlls, this is mainly so people running pilot visor colour mods can keep those, since they use a client.dll edit
\ No newline at end of file
+host_skip_client_dll_crc  1 // allow people to run modded client dlls, this is mainly so people running pilot visor colour mods can keep those, since they use a client.dll edit
+
+announcementVersion 1
+announcement #PROGRESSION_ANNOUNCEMENT_BODY
```

Comment:
I'm ootl, what does this do?

Reply:
This sets the announcement body text for this announcement (vanilla thing, just we didnt implement a little bit of it)
the announcement version is used to ensure the user doesn't see the same announcement multiple times.
![image](https://github.com/R2Northstar/NorthstarMods/assets/66967891/3fa71511-3e65-4cf5-b63a-e0a474df7bfe)

In future i'd like to get this convar from Atlas, similarly to mainmenupromos but this will do for now.